Dockerfile

```diff
@@ -22,20 +22,24 @@ RUN apk add --update \
     curl-dev \
  && rm -rf /var/cache/apk/* \
  && bundle config --global frozen 1 \
- && bundle install --without test production --jobs 2 \
+ && bundle install --without test --jobs 2 \
  && gem install foreman
 
 # ========================================================
 # Application layer
 
 # Copy application code
-COPY . /usr/src/app
+COPY . $APP_ROOT
+
+# Precompile assets for a production environment.
+# This is done to include assets in production images on Dockerhub.
+RUN RAILS_ENV=production bundle exec rake assets:precompile
```

Comment:
Will this work if you’ve not got a production config (I.e database)? Would it still work without the RAILS_ENV?

Reply:
> Will this work if you’ve not got a production config (I.e database)?

Yup I believe so.

> Would it still work without the RAILS_ENV?

AFAIK no. From what I understand assets are keyed per environment; in fact I've seen that only way to do something such as including precompiled assets for use in any environment is to literally repeat this command for all three envs 😧